hello.py

```diff
@@ -1 +1,2 @@
-print "Hello World!"
+print "Hello World!  ver 2.0."
```

Comment:
This just in from marketing: we really need to call it 5.0 to keep up with the competition. Can you please change?